### DISCUSSION TITLE
Slow down the voice?

### DISCUSSION THREAD

**Discussion by gexagonr** [neonbjb/tortoise-tts#211] [General]
Is there a way to make the pace of the voice slower? Seems to talk a little too fast for me by default.

**Comment by neonbjb** [comment]
You could try using a conditioning clip that speaks at a slower rate,
although people have only had mixed results with that.

You can also just resample
<https://pytorch.org/audio/stable/tutorials/audio_resampling_tutorial.html>the
audio at a higher sample rate, but save it at the default rate.

On Mon, Dec 5, 2022 at 1:26 AM kpakuart ***@***.***> wrote:

> Is there a way to make the pace of the voice slower? Seems to talk a
> little too fast for me by default.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/neonbjb/tortoise-tts/discussions/211>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGLMOTXRFOKIBBI6IQXKOTWLWRKRANCNFSM6AAAAAAST7IOWU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
-
James Betker

**Reply by gexagonr** [reply]
Sorry, can you give more details on resampling, at a higher sample rate, and saving it at default rate? Im not too knowledgable on audio engineering. But thanks for the reply!

**Comment by ThioJoe** [comment]
@neonbjb Also referencing discussion #95 , is this a limitation of the technique / model itself? Or would it be possible to add the feature in the future? Like how Google and Azure TTS services allow you to enter in a float number to adjust the speaking speed.

**Comment by neonbjb** [comment]
I'm sure there's a way to add a feature like this, but I am not actively developing this project. Sorry!

**Reply by fitbyprabhat** [reply]
Neon bro what u did is already a wonder

**Comment by manmay-nakhashi** [comment]
now you can control speaking rate :smile: :rocket:

**Reply by totomtornado** [reply]
How?
